configs/roles/remediations.json

```diff
@@ -27,6 +39,18 @@
         },
         {
           "permission": "remediations:remediation:write"
+        },
+        {
+          "permission": "playbook-dispatcher:run:read",
```

Comment:
@jharting it looks like this is identical to the entry above, is this supposed to be `"permission": "playbook-dispatcher:run:write",` here?

Reply:
@coderbydesign 
No, it's on purpose. I was trying to make the roles self-contained i.e. if I assign a user to `Remediations administrator` they should get full access even without having `Remediations user` (i.e. it should not be assumed that the administrator already has read permission from the Remediations user role). LMK if that's wrong.

Reply:
No that makes sense. I was reading this permission initially as being a part of the `Remediations administrator` role because of the way GH had the diff. 😄 

You will however want to bump the version of this role in order to pick the changes up.

Reply:
@coderbydesign good catch. It should be fixed now.